docs/install.md

```diff
@@ -936,3 +936,155 @@ This procedure can be used to downgrade the active version of SAT.
+  documentation for more information.
+
+### Procedure
+
+1. Copy the file `/etc/kubernetes/admin.conf` from `ncn-m001` to `~/.kube/config` on the external system.
+
+    ```
+    $ mkdir -p ~/.kube
+    $ scp ncn-m001:/etc/kubernetes/admin.conf ~/.kube/config
+    admin.conf                                       100% 5566   3.0MB/s   00:00
+    ```
+
+1. Add an new entry for the hostname `kubernetes` to the external system's `/etc/hosts` file.
+
+   The `kubernetes` hostname should correspond to the CAN IP address on `ncn-m001`. On CSM 1.2, this can be determined
+   by querying the IP address of the `bond0.cmn0` interface.
+
+   ```
+   $ ssh ncn-m001 ip addr show bond0.cmn0
+   13: bond0.cmn0@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether b8:59:9f:1d:d9:0e brd ff:ff:ff:ff:ff:ff
+    inet 10.102.1.11/24 brd 10.102.1.255 scope global vlan007
+       valid_lft forever preferred_lft forever
+    inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+       valid_lft forever preferred_lft forever
+   $ IP_ADDRESS=10.102.1.11
+   ```
+
+   On CSM versions prior to 1.2, the CAN IP can be determined by querying the IP address of the `vlan007` interface.
+
+   ```
+   $ ssh ncn-m001 ip addr show vlan007
+   13: vlan007@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether b8:59:9f:1d:d9:0e brd ff:ff:ff:ff:ff:ff
+    inet 10.102.1.10/24 brd 10.102.1.255 scope global vlan007
+       valid_lft forever preferred_lft forever
+    inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+       valid_lft forever preferred_lft forever
+   $ IP_ADDRESS=10.102.1.10
+   ```
+
+   Once the IP address is determined, add an entry to `/etc/hosts` mapping the IP address to the hostname `kubernetes`.
+
+    ```
+    $ echo "${IP_ADDRESS} kubernetes" | sudo tee -a /etc/hosts
+    10.102.1.11 kubernetes
+    ```
+
+1. Modify `~/.kube/config` to set the cluster server address.
+
+   The value of the `server` key for the `kubernetes` cluster under the `clusters` section should be set to
+   `https://kubernetes:6443`.
+
+    ```yaml
+    ---
+    clusters:
+    - cluster:
+        certificate-authority-data: REDACTED
+        server: https://kubernetes:6443
+      name: kubernetes
+    ...
+    ```
+
+1. Confirm that `kubectl` can access the CSM Kubernetes cluster.
+
+    ```
+    $ kubectl get nodes
+    NAME       STATUS   ROLES    AGE    VERSION
+    ncn-m001   Ready    master   135d   v1.19.9
+    ncn-m002   Ready    master   136d   v1.19.9
+    ncn-m003   Ready    master   136d   v1.19.9
+    ncn-w001   Ready    <none>   136d   v1.19.9
+    ncn-w002   Ready    <none>   136d   v1.19.9
+    ncn-w003   Ready    <none>   136d   v1.19.9
+    ```
+
+1. Use `sat init` to create a configuration file for SAT.
+
+    ```
+    $ sat init -o /home/user/.config/sat/sat.toml
```

Comment:
When I was the `root` user on my VM running with the podman wrapper script, I did not need `-o`, and I think it actually would have been wrong for that use case. I think it would also be unnecessary when running in a python virtualenv.